superset-frontend/src/components/EditableTitle/index.tsx

```diff
@@ -18,8 +18,9 @@
  */
 import React, { useEffect, useState, useRef } from 'react';
 import cx from 'classnames';
-import { t } from '@superset-ui/core';
+import { styled, t } from '@superset-ui/core';
 import { Tooltip } from 'src/components/Tooltip';
+import CertifiedIcon from '../CertifiedIcon';
```

Comment:
This component probably shouldn't even exist, but that's a different discussion ;)

Reply:
@rusackas I think the name is misleading here. This is intended to be more than just an icon, it's more like a tiny component.

Reply:
Then maybe we can change the name to CertifiedBadge to avoid confusion?

Reply:
Yes. I will open a separate PR